news/2025/2025-05-18-mappers-choice-awards-2024-results.md

```diff
@@ -0,0 +1,103 @@
+---
+layout: post
+title: "Mapper's Choice Awards 2024: Results"
+date: 2025-05-18 20:30:00 +0000
+---
+
+The mappers have chosen! Let's comb through the results of this year's iteration of the Mapper's Choice Awards!
+
+![](/wiki/shared/news/2025-03-26-mappers-choice-awards-2024/banner.jpg)
+
+In case you aren't up to date with the happenings of the mapping scene, the Mappers' Choice Awards (or *MCA*, as it is most known) is a community-hosted event where mappers nominate and vote on maps and mappers for various categories in order to celebrate mapping excellence in all shapes and forms. And at last, we get to commemorate the best of this year in categories of each game mode (plus storyboarding)!
+
+While we'll only be covering the Grand Award podiums in this post, [all category rankings can be found in the MCA website](https://mca.corsace.io/2024/results), so make sure to not miss out on the full results there!
+
+## Rewards
+
+Before we dive into the results, here's a quick refresher on what awards we're offering this year:
+
+- **osu!supporter**
+  - Grand Award winners: 4 months
+- **Badges**
+  - Winners of the Grand categories will be awarded special badges each.
+  - 2nd and 3rd place of the Grand categories will be awarded a top 3 badge.
```

Comment:
this is slightly different from the announcement https://osu.ppy.sh/home/news/2025-03-26-mappers-choice-awards-2024, so "refresher" isn't entirely accurate

```suggestion
The winners presented here will find themselves with the following rewards:

- **osu!supporter**
  - Grand Award winners: 4 months
- **Badges**
  - Winners of the Grand categories will be awarded special badges each.
  - 2nd and 3rd place of the Grand categories will be awarded a top 3 badge.

Due to comparatively low participation this year, the amount of badge awards has been reduced to only Grand category winners.
```